Clother/View Models/ClothesViewModel.swift

```diff
@@ -26,59 +26,120 @@ class ClothesViewModel {
     }
     
     // TODO: Creates an outfit item and stores it in our application.
-    func createOutfit(top: ClothingItem, bottom: ClothingItem, shoes: ClothingItem, other: [ClothingItem]) {
-        
+    func createOutfit(name: String, top: ClothingItem, bottom: ClothingItem, shoes: ClothingItem, other: [ClothingItem]) {
+        let outfit = OutfitItem(name: name, top: top, bottom: bottom, shoes: shoes, other: other)
+        userOutfits.append(outfit)
     }
     
     // TODO: Creates a clothing item and stores it in our application.
     func createClothing(name: String, image: Data, type: Clothing.Kind, size: Clothing.Size, color: Clothing.Color, weather: Clothing.Weather, fabric: Clothing.Fabric) {
-        
+        let clothing = ClothingItem(name: name, type: type, size: size, color: color, weather: weather, fabric: fabric, displayImage: image)
+        userClothes.append(clothing)
     }
     
     /* TODO: Randomly creates an outfit out of the provided constraints and returns it. Do not store it anywhere.
-        Instead of an OutfitItem, you will return a tuple of 3 values in the form of (top, bottom, shoes, accessories). If no top/bottom/shoes are available on the app, return nil.
-        If a provided constraint is nil, it will not be enforced. HINT: you can utilize createOutfit(...) to avoid repeating code.
-        NOTE: leave the name to be empty.
+     Instead of an OutfitItem, you will return a tuple of 3 values in the form of (top, bottom, shoes, accessories). If no top/bottom/shoes are available on the app, return nil.
+     If a provided constraint is nil, it will not be enforced. HINT: you can utilize createOutfit(...) to avoid repeating code.
+     NOTE: leave the name to be empty.
      */
-    func generateRandomOutfit(type: Clothing.Kind?, size: Clothing.Size?, color: Clothing.Color?, weather: Clothing.Weather?, fabric: Clothing.Fabric?)
+    func generateRandomOutfit(size: Clothing.Size?, color: Clothing.Color?, weather: Clothing.Weather?, fabric: Clothing.Fabric?)
```

Comment:
I think the inputs we take for generating a random outfit have some redundancy. How about this, for size all clothing items must of the given size. For Color, only one of (top, bottom) must be of that color. For weather they must all be suited for that weather.  For fabric, only of (top, bottom) must be of that color.